Error when try to import ClayMultiStep React component

Hi, guys!!
When I try to import **ClayMultiStepNav** componente, an error is displayed on console:

```
ERROR in ./node_modules/@clayui/multi-step-nav/lib/MultiStepNavWithBasicItems.js
Module not found: Error: Can't resolve '../src/MultiStepNav' in '/Users/interaminense/workspace/analytics/com-liferay-osb-faro-private/osb-faro-web/node_modules/@clayui/multi-step-nav/lib'
 @ ./node_modules/@clayui/multi-step-nav/lib/MultiStepNavWithBasicItems.js 8:43-73
 @ ./node_modules/@clayui/multi-step-nav/lib/index.js
 @ ./src/main/js/experiments/components/summary-card-draft/Body.tsx
 @ ./src/main/js/experiments/components/summary-card-draft/index.tsx
 @ ./src/main/js/experiments/hocs/SummaryCard.tsx
 @ ./src/main/js/experiments/pages/ExperimentOverviewPage.tsx
 @ ./src/main/js/shared/pages/AppSidebarRoutes.jsx
 @ ./src/main/js/App.jsx
 @ ./src/main/js/main.jsx
 @ multi (webpack)-dev-server/client?http://localhost:3000 ./src/main/js/main.jsx
```

Someone can help me with it?

--- a/packages/clay-multi-step-nav/src/MultiStepNavWithBasicItems.tsx
+++ b/packages/clay-multi-step-nav/src/MultiStepNavWithBasicItems.tsx
@@ -4,7 +4,7 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-import ClayMultiStepNav from '../src/MultiStepNav';
+import ClayMultiStepNav from './MultiStepNav';
 import React from 'react';
 import {ClayDropDownWithBasicItems} from '@clayui/drop-down';
 

